java/flight/flight-core/src/main/java/org/apache/arrow/flight/FlightConstants.java

```diff
@@ -23,5 +23,6 @@
 public interface FlightConstants {
 
   String SERVICE = "arrow.flight.protocol.FlightService";
+  String SESSION_HEADER = "SET-SESSION";
```

Comment:
Is there a particular reason why this is `SET-SESSION` as opposed to `Set-Session`? `Set-Session` would be more in line with other headers being used, such as `Set-Cookie, Authorization or Arrow-Properties`

Reply:
Addressed.